harness/determined/_trial_controller.py

```diff
@@ -101,11 +100,6 @@ def run(self) -> None:
     def supports_mixed_precision(cls: Type["TrialController"]) -> bool:
         return False
 
-    @classmethod
-    @abc.abstractmethod
-    def create_metric_writer(cls: Type["TrialController"]) -> tensorboard.BatchMetricWriter:
-        pass
-
```

Comment:
fun fact: you could technically restore this now that Anda landed the Trainer API, which actually pulls everything PyTorchTrial-related out of the old Trial and TrialController type hierarchy.

I don't suggest that you bother doing that, because the only two actually affected objects are EstimatorTrial and TFKerasTrial and I don't think they're worth the effort, _at all_.  I point this out to give you an idea of how rigid the old patterns we used are and to illustrate how they're being broken apart as we move forward.